core/util/src/main/scala/net/liftweb/util/Props.scala

```diff
@@ -328,5 +378,18 @@ object Props extends Logger {
         Map()
     }
   }
+
+  private var prepended:List[java.util.Map[Object, Object]] = Nil
+  private var appended:List[java.util.Map[Object, Object]] = Nil
+  private var interpolators:List[java.util.Map[Object, Object]] = Nil
```

Comment:
I got really excited about this idea, but [the scaladoc for `scala.sys`](http://scala-lang.org/files/archive/api/current/#scala.sys.package) shows that `props` is a `SystemProperties` which is a `mutable.Map`. Regardless, I feel a lot better about using a scala collection here, but still not too keen on making them immutable. I would be afraid that someone would be confused if the java props got changed and not reflected in the Lift `Props`.  What do you think, @farmdawgnation?